.changeset/quiet-peaches-film.md

```diff
@@ -0,0 +1,9 @@
+---
+'@apollo/cache-control-types': patch
+'@apollo/server-gateway-interface': patch
+'@apollo/server-plugin-response-cache': patch
+'@apollo/server': patch
+'@apollo/usage-reporting-protobuf': patch
+---
+
+Publish TypeScript typings for CommonJS modules
```

Comment:
Would you mind adding just a bit more context here? For users reading this - what problem exactly does it solve for them? What configurations were previously broken that this change unlocks?

Reply:
Added a bit more meat, I hope this is enough, I'm really not great when it comes to writing changelog entries

Reply:
Works for me!